internal/store/builder.go

```diff
@@ -203,11 +203,21 @@ func (b *Builder) WithAllowAnnotations(annotations map[string][]string) {
 func (b *Builder) WithAllowLabels(labels map[string][]string) error {
 	if len(labels) > 0 {
 		for label := range labels {
-			if !resourceExists(label) {
+			if !resourceExists(label) && label != "*" {
 				return fmt.Errorf("resource %s does not exist. Available resources: %s", label, strings.Join(availableResources(), ","))
 			}
 		}
 		b.allowLabelsList = labels
+		// "*" takes precedence over other specifications
+		if allowedLabels, ok := labels["*"]; ok {
+			m := make(map[string][]string)
+			for _, resource := range b.enabledResources {
+				m[resource] = allowedLabels
+			}
+			b.allowLabelsList = m
+		} else {
```

Comment:
Is this `else` path needed? It's already set before in line 210

Reply:
Must've got in while resolving conflicts during the rebase, fixed.